litmus-portal/graphql-server/pkg/cluster/handler/handler.go

```diff
@@ -175,6 +180,7 @@ func DeleteClusters(ctx context.Context, projectID string, clusterIds []*string,
 	if err != nil {
 		return "", err
 	}
+
 	clusters, err := dbOperationsCluster.ListClusters(ctx, query)
```

Comment:
This function should ideally not return any clusters, since we are updating the documents with `is_removed: true` and then we are querying the same documents with `is_removed: false`